pkg/graph/schema/types/translation.graphql

```diff
@@ -0,0 +1,62 @@
+"""
+Base typea that represents FE translation structure
```

Comment:
Is this a block comment for the file?  that comment structure attaches as a description in the GQL Schema. 
(in this case to the TranslationDataType). Could you change the page comment to use `#` at the start of every line instead?

if you update the type comments to use `"""`, they will be exposed in the GQL introspection query

Reply:
Gotcha, thanks I have no idea how comments work here.  I just copied from the other related file haha